### PR TITLE
[fix] Copy test files from new app template

### PR DESCRIPTION
**Pull request by tnunes**
Complements https://github.com/trailsjs/trails/pull/207.

### CODE REVIEW

File: src/app/steps/writing.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ export default {
   genericApi () {
     this.fs.copy(path.resolve(TRAILS_TEMPLATE, 'api/services', '**'), this.destinationPath('api/services'))
     this.fs.copy(path.resolve(TRAILS_TEMPLATE, 'api/models', '**'), this.destinationPath('api/models'))
+    this.fs.copy(path.resolve(TRAILS_TEMPLATE, 'test', '**'), this.destinationPath('test'))
+    this.fs.copy(path.resolve(TRAILS_TEMPLATE, 'test', '.*'), this.destinationPath('test'))
```
Comment (jaumard) [inline src/app/steps/writing.js]: this two line should be move under `root()` as this method is for `api` files :)

Comment (tnunes) [inline src/app/steps/writing.js]: Good catch :) Done in f91fa18.

**Comment by jaumard** [comment]
@tnunes you need to synchronize your branch please

**Comment by tnunes** [comment]
@jaumard done! Any progress with this PR and https://github.com/trailsjs/trails/pull/207?

**Comment by jaumard** [comment]
Thanks I'll merge it when your other PR will be and tests were green ^^. Don't know for the other PR doesn't depends on me sorry

**Comment by tnunes** [comment]
The tests should now pass, since https://github.com/trailsjs/trails/pull/207 was already merged.

Can we re-trigger the checks somehow without having to commit new changes?

**Comment by jaumard** [comment]
@tnunes it doesn't pass because your trails PR is merged but it was not deployed on npm so your change are not yet available on this module

**Pull request merged**